net/http/src/TRootSniffer.cxx

```diff
@@ -389,7 +388,7 @@ TRootSnifferScanRec::GoInside(TRootSnifferScanRec &super, TObject *obj, const ch
       SetField("_fullname", full_name);
 
    if (topelement)
-      SetField(item_prop_rootversion, TString::Format("%d", ROOT_VERSION_CODE), kFALSE);
+      SetField(item_prop_rootversion, TString::Format("%d", gROOT->GetVersionCode()), kFALSE);
```

Comment:
Shouldn't you then include `TROOT.h` to use `gROOT`?

Reply:
`TROOT.h` is already included - there are several other places with `gROOT`